api/pkg/apis/v1alpha1/model/deployment.go

```diff
@@ -25,6 +25,7 @@ type DeploymentSpec struct {
 	Generation          string                 `json:"generation,omitempty"`
 	ObjectNamespace     string                 `json:"objectNamespace,omitempty"`
 	Hash                string                 `json:"hash,omitempty"`
+	IsDryRun            bool                   `json:"isDryRun,omitempty"`
```

Comment:
I see you access the `dryRun` in Reconcile. Should be a default value, remove omitempty?

Reply:
the "omitempty" option is used to omit a field from the JSON output if it contains an empty value or a zero.
If you access the field in deploymentSpec, you can always get the value regardless of omitempty or not. And if the Json string doesn't contains isDryRun, the result of unmarshall will have false IsDryRun by default.